python/tskit/trees.py

```diff
@@ -8478,8 +8484,12 @@ def genetic_relatedness_weighted(
         centre=True,
     ):
         r"""
-        Computes weighted genetic relatedness. If the :math:`k` th pair of indices
-        is (i, j) then the :math:`k` th column of output will be
+        Computes weighted genetic relatedness.
+        Please see the :ref:`summary functions <sec_stats_summary_functions>`
+        section on the exact definition of the calculated statistic.
```

Comment:
I don't actually think the "summary function" is a good place to refer people to here. That is a definition, but it only makes sense if someone understands How It All Works. The approach we've taken in other statistics is to provide a more plain (but equivalent) definition. I think here the thing to do is to just refer to the docs for `genetic_relatedness`, because this function's notion of genetic relatedness is the same as that one.